guildwars2/characters.py

```diff
@@ -388,6 +401,33 @@ def handle_duplicates(upgrades):
         if character == "All":
             await user.send("\n".join(output))
 
+    @character.command(name="crafting")
+    @commands.cooldown(1, 10, BucketType.user)
+    async def character_crafting(self, ctx):
+        """Displays your characters and their crafting level"""
+        endpoint = "characters?page=0"
+        await ctx.trigger_typing()
+        try:
+            characters = await self.call_api(endpoint, ctx.author,
+                                             ["characters"])
+        except APIError as e:
+            return await self.error_handler(ctx, e)
+        data = discord.Embed(
+            description='Crafting overview', colour=self.embed_color)
```

Comment:
Embed could be fancified a bit - like getting the doc via `fetch_key()` before, then doing
```py
embed.set_author(name=doc["account_name"], icon_url=ctx.user.avatar_url)
```

Reply:
Implemented, but should be ctx.author.avatar_url